pkg/generate/app/cmd/newapp.go

```diff
@@ -222,26 +224,56 @@ func (c *AppConfig) SetOpenShiftClient(osclient client.Interface, OriginNamespac
 // AddArguments converts command line arguments into the appropriate bucket based on what they look like
 func (c *AppConfig) AddArguments(args []string) []string {
 	unknown := []string{}
+	c.ArgumentClassificationErrors = map[string]error{}
 	for _, s := range args {
-		switch {
```

Comment:
actually, the refactor into functions made me realize I could preserve the switch - thanks!... part of next push